hydra-http-form.c

```diff
@@ -760,10 +861,29 @@ int32_t start_http_form(int32_t s, char *ip, int32_t port, unsigned char options
   clogin[sizeof(clogin) - 1] = 0;
   strncpy(cpass, html_encode(pass), sizeof(cpass) - 1);
   cpass[sizeof(cpass) - 1] = 0;
-  upd3variables = hydra_strrep(variables, "^USER^", clogin);
-  upd3variables = hydra_strrep(upd3variables, "^PASS^", cpass);
-  upd3variables = hydra_strrep(upd3variables, "^USER64^", b64login);
-  upd3variables = hydra_strrep(upd3variables, "^PASS64^", b64pass);
+
+  if (multipart_mode) {
+    char multipart_boundary[32] = "----THC-HydraBoundaryz2Z2z";
+
+    snprintf(content_type, sizeof(content_type), "multipart/form-data; boundary=%s", multipart_boundary);
```

Comment:
is there a reason you are not doing directly: `snprintf(content_type, sizeof(content_type), "multipart/form-data; boundary=----THC-HydraBoundaryz2Z2z");` ?
of just strcpy because we know the size is fine?